integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -112,3 +99,23 @@ def transform_end(self, content_metadata_item):
         if end_date:
             return int(parse(end_date).timestamp())
         return None
+
+    def transform_announcement(self, content_metadata_item):
+        """
+        Formats a post in a course's Announcement forum with
+        edX's course summary and enrollment link
+        """
+        full_description = content_metadata_item.get('full_description') or None
+        short_description = content_metadata_item.get('short_description') or None
+
+        if full_description and len(full_description) <= self.LONG_STRING_LIMIT:
+            description = full_description
+        else:
+            description = short_description
```

Comment:
Is there always a short description to be relied on? Or does a truncation of the long description need to exist as a fallback?

Reply:
I believe in practice short_description is always present. In one of these I had done title as a fallback if none of these were present and valid, but found title to be less often present than even short_description.

Reply:
Arbitrarily chopping up the full_description was determined to be a worse UI experience than using a short_description.